topics/03_file_handling_and_regex/Regular Expressions.py

```diff
@@ -0,0 +1,42 @@
+# Define if a string contains the required characters. E.g. if '7865serS3' includes '583' - True; '973' - False
+import re
+
+def contains_chars(main_string, required_string):
+
+    pattern = ''.join([f'(?=.*{char})' for char in required_string])
+
+    # Search the main_string using the pattern
+    if re.search(pattern, main_string):
+        return True
+    else:
+        return False
+
+# test
+print(contains_chars('7865serS3', '583'))  # -> True
+print(contains_chars('7865serS3', '973'))  # -> False
+
+#Count a number of Upper case letters in the string. E.g. '7865serS3' - 'Number of Capital letters: 1'
+import re
```

Comment:
No need in duplicated imports.
Good approach to have them all at the top of the file